src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.pscdataapi.api;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+
+@ExtendWith(MockitoExtension.class)
+class ResourceChangedApiServiceAspectTest {
```

Comment:
The class name has "Aspect", while the filename does not. They should be the same